core/controller/src/main/scala/org/apache/openwhisk/core/controller/Actions.scala

```diff
@@ -212,11 +212,13 @@ trait WhiskActionsApi extends WhiskCollectionAPI with PostActionActivation with
     parameter('overwrite ? false) { overwrite =>
       entity(as[WhiskActionPut]) { content =>
         val request = content.resolve(user.namespace)
-        val checkAdditionalPrivileges = entitleReferencedEntities(user, Privilege.READ, request.exec).flatMap {
-          case _ => entitlementProvider.check(user, content.exec)
-        }
+        val check = for {
+          checkLimits <- checkNamespaceAndSystemLimits(user, content)
```

Comment:
For this kind check, the limit check should be checked under the action's activation is executing?
Why when create action, should check this kind limit?

Reply:
@ningyougang 
Sorry, I don't understand. It is an existing spec to check the limit when creating an action.
You still can't create an action that exceeds the system limit.

And this method is to check the limit of the namespace along with the system limit.

Reply:
Got it